recipe/meta.yaml

```diff
@@ -57,6 +58,7 @@ requirements:
     - xorg-libxfixes
     - xorg-glproto
     - xorg-libxdamage
+    - llvm
```

Comment:
do you need llvm here? llvm should export its own dependencies

Reply:
Oh I wasn't aware. Removed.